sonar-batch-protocol/src/main/gen-java/org/sonar/server/source/db/FileSourceDb.java

```diff
@@ -4139,6 +4167,7 @@ public boolean hasMsg() {
       }
     }
 
+    // repeated .org.sonar.server.source.db.Test.CoveredFile covered_file = 7;
     public static final int COVERED_FILE_FIELD_NUMBER = 7;
     private java.util.List<org.sonar.server.source.db.FileSourceDb.Test.CoveredFile> coveredFile_;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "coveredFile_" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)